migra/changes.py

```diff
@@ -2,11 +2,16 @@
 
 from collections import OrderedDict as od
 from functools import partial
+from venv import create
 
 import schemainspect
 
-from .statements import Statements
-from .util import differences
+try:
```

Comment:
this is highly bizarre, it looks to me like you wanted to both test modules as you were writing them (non-relative) and but use the relative imports when the package was used as a library

potentially fine for debugging + local workflow, but would never push this and would definitely not merge this

Reply:
Yeah, just for the debugging workflow. Agree with the analysis